### PR TITLE
Techdebt/363 update filemanager index

### PR DESCRIPTION
**Pull request by ruilealsilva**

### CODE REVIEW

**Comment by fericsepi** [comment]
Issue #363

File: frontend/src/pages/FileManager/index.js [hunk 1]
```diff
@@ -91,106 +91,98 @@ const Loading = () => (
...
-    this.props.files.length <= 0 &&
-      !this.props.fileDeleted &&
-      (await this.props.loadFiles());
-    //view files when attributes passed from Email link
-    if (this.props.location.params && this.props.location.params.attachment) {
-      this.props.location.params.attachment.fileUrl &&
-        this.displayFileModalHandler(
-          this.props.location.params.attachment.fileUrl
-        );
+const FileManager = props => {
```
Comment (bendiggle) [inline frontend/src/pages/FileManager/index.js]: We only use location from props so could use object destructuring here to make it cleaner
```suggestion
const FileManager = ({ location }) => {
```

**Comment by bendiggle** [comment]
You have some linting errors
Screenshot 2019-11-26 at 12 55 32

**Comment by bendiggle** [comment]
@ruilealsilva There's a bug where if you view an attachment from an email (it takes you to the files page) but if you then delete a file, that attachment appears straight after every time

File: frontend/src/pages/FileManager/index.js (hunk 1 above)
Comment (dsantoro) [reply frontend/src/pages/FileManager/index.js]: So @bendiggle  came to the destructuring side of the force.

**Pull request merged**